libs/ECS/Components/ControlableComponent.hpp

```diff
@@ -0,0 +1,33 @@
+/*
+** EPITECH PROJECT, 2022
+** Project
+** File description:
+** ControlableComponent
+*/
+
+#ifndef CONTROLABLECOMPONENT_HPP_
+#define CONTROLABLECOMPONENT_HPP_
+
+#include "Component.hpp"
+
+namespace ecs
+{
+    /// @brief The controlable class, defining if an entity is affected by the player's input
+    class Controlable : public Component {
```

Comment:
I think that this component should be empty. The fact that this component is linked to an entity marks it as an controllable entity